app/src/pages/movies/filter/Filter.jsx

```diff
@@ -0,0 +1,26 @@
+import { FilterListComponent } from './components/layouts/FilterListComponent';
+import { FilterOption } from './components/FilterOption';
+
+const options = [
+    'ALL', 'DOCUMENTARY', 'COMEDY', 'HORROR', 'CRIME'
+]
+
+// TBD
+let choosed = option => option == options[0];
```

Comment:
I guess `chosen`)

we usually use `isActive` `isSelected` `selected` and so on

Reply:
Fixed)